src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/expectedInputJavadocComment.txt

```diff
@@ -1,25 +1,25 @@
-JAVADOC -> <p>\r\nMy {@code Class}\r\n</p>\r\n@see never<EOF> [0:0]
```

Comment:
yes, module that create fake stuff(wrapping) should be ready to correct Tree to expected values.